src/welcome.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+const Welcome = ({ account, balance, style={} }) => {
+  
+  const buttonContainer = {
```

Comment:
Move these style object literal declarations outside of `render` method to avoid having to allocate everytime `render` is called. You can move these outside of the component itself into the module.